### DISCUSSION TITLE
请问迅雷现在如何添加？之前成功过一次，过期后就再也不行了。

### DISCUSSION THREAD

**Discussion by curryhendry** [AlistGo/alist#6177] [General]
ErrorCode: 4002 ,Error: captcha_invalid ,ErrorDescription: client_id:"Xp6vsxz_7IYVw2BB" meta:{key:"phone_number" value:"+86 18********esult" value:"review"} result:review

**Comment by phcl1998** [comment]
一样

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Reply by curryhendry** [reply]
这不就是标准添加的方法吗？不是这个现象。

**Reply by kaixinxiaowangzi666** [reply]
看看就知道

**Reply by curryhendry** [reply]
谢谢您的回复，我说看过才说的，确实不是一个问题。这就是教程里的操作方法我第一次也是这样添加的。

**Comment by feicun2** [comment]
+1

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Comment by lianhaisen** [comment]
+1

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5

**Comment by wow8szh** [comment]
我刚添加成功，+86后面加个空格就解决了。我的问题是添加后状态正常，文件列表也都正常，就是不能播放里面的电影文件。

**Reply by kaixinxiaowangzi666** [reply]
【【alist挂载迅雷云盘出现问题，已有解决办法了】】 https://www.bilibili.com/video/BV1tm411R7aP/?share_source=copy_web&vd_source=c704f767b91c2b806257c094a6ec9fc5
